cookiecutter.json

```diff
@@ -1,7 +1,9 @@
 {
     "driver_name": "MyDriver",
     "driver_name_lc": "{{ cookiecutter.driver_name.lower() }}",
-    "release": "queens",
+    "release": "ussuri",
     "additional_package_name": "",
-    "driver_install_method": ["None", "PPA", "Juju Resource DEB"]
+    "driver_install_method": ["None", "PPA", "Juju Resource DEB"],
+    "stateless": [true, false],
+    "active_active": [true, false]
```

Comment:
As far as I understand, the cookiecutter cannot show a brief explanation what the config is for. So putting stateless/active_active here may be confusing. I would suggest dropping those from here, but to put into the class with comments when those options should be `true`.

Reply:
Indeed, there's no way to add descriptions. The idea of putting them here was to make sure the users knew about these parameters so they'd have to consciously think about them.

Reply:
This is true, but this could probably be spelled out a bit more clearly in the readme.